MovieApp/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+
+import UIKit
+import PureLayout
+
+class MyCollectionViewCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "crewCellIdentifier"
+    
+    private let crewMemberNameLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    private let crewMemberPositionLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        contentView.addSubview(crewMemberNameLabel)
+        contentView.addSubview(crewMemberPositionLabel)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .top, withInset: 3)
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
```

Comment:
What would happen if the crew member has a long name?

The name would go out of bounds of this cell. Make sure to always have all constraints set up (leading, trailing, top, bottom).